app/src/main/java/net/sourceforge/opencamera/PreferenceHandler.java

```diff
@@ -0,0 +1,96 @@
+package net.sourceforge.opencamera;
+
+import android.content.SharedPreferences;
+import android.preference.PreferenceManager;
+import android.util.Log;
+
+public class PreferenceHandler {
```

Comment:
Naming is not great in my opinion, because handler is associated with multithreading (handler and looper from android). It will be better to use controller, service or repository word. Also all boolean getters should be renamed like getIMURecordingPref -> isEMURecording() or maybe with word  isEMURecordingEnabled if it is appropriate for this case.

Reply:
We tried to match the global style, e.g. PermissionHandler